activities/FractionBounce.activity/js/activity.js

```diff
@@ -295,11 +303,16 @@ let app = new Vue({
 				slopeCanvas.removeEventListener("mousedown", this.onTouchStart);
 				this.onTouchEnd();
 				let result = this.$refs.slopecanvas.checkAnswer();
+				count++;
 				if (result == this.answer) {
 					this.successSound.play();
+					score++;
 				} else {
 					this.failSound.play();
 				}
+				if (count < 30) {
```

Comment:
Why 30? Not sure to understand.

Reply:
I tried printing count for each question in this method. But as this method is called trice for each question I got count 3 in console. So I have taken count<30 i.e till 10 questions  and when 10 questions are reached and then when we click on play button then as the new turn begins i changed score to 0/10